source/vir/src/def.rs

```diff
@@ -223,25 +220,28 @@ pub fn trait_self_type_param() -> Ident {
     Arc::new(TRAIT_SELF_TYPE_PARAM.to_string())
 }
 
-pub fn height() -> Fun {
-    Arc::new(FunX {
-        path: Arc::new(PathX { krate: None, segments: Arc::new(vec![str_ident(HEIGHT)]) }),
-        trait_path: None,
-    })
+pub fn height() -> Node {
+    str_to_node(&height_ident())
 }
 
-pub fn closure_req() -> Fun {
-    Arc::new(FunX {
-        path: Arc::new(PathX { krate: None, segments: Arc::new(vec![str_ident(CLOSURE_REQ)]) }),
-        trait_path: None,
-    })
+pub fn height_ident() -> Ident {
```

Comment:
> Actually, looking over def.rs, I think PREFIX_STR and PREFIX_CHAR are not necessary, because they are only used for a fixed set of constant identifiers. Where we need prefixes is for names that are combined with other names, like PREFIX_FUEL_ID. So we should be able to get rid of functions like strslice_is_ascii_ident, too.

This make sense, but I'll make this fix separately from this PR.